### ISSUE TITLE
Project page's demo instructions don't work

### ISSUE THREAD

**Issue by wimleers** [tonystar/drupal-atoms#26]
> 1. Open this link (pre-configured sandbox on simplytest.me).
> 2. Click Launch sandbox and wait.
> 3. Click Log in (email and password should be pre-filled).
> 4. Go to Modules and enable Atoms Default.

Step 4 does not work: that module is not present.

**Comment by tonystar** [comment]
It was renamed to **Atoms Basic** since there will be several submodules providing default atoms.

**Comment by wimleers** [comment]
Okay, that worked :)

**Comment by wimleers** [comment]
But… once I've done that, "Create group" and "Create columns" seem broken?

**Comment by tonystar** [comment]
No! They works! This is layout-only atoms (without any data). That's why the form does not contain fields.

**Comment by tonystar** [comment]
To render atoms in columns just reorder them as childs when editing the node! (that could be buggy, but after re-save it should work).

**Comment by OnkelTem** [comment]
Hi there! Привет!

I personally wished to start with screencasts, but as I've found none, I tried simpetest.me demo and it worked for me so I managed to create my first atomic page! So please treat this comment as a vote for cooking some useful screencasts.
